client/src/components/NavBar/NavBar.tsx

```diff
@@ -22,6 +22,9 @@ interface Props {
 const NavBar = ({ loggedInUser, handleDrawerToggle }: Props): JSX.Element => {
   const classes = useStyles();
 
+  // Need to grab the active board from context and use it's information here instead
+  // Title, anything else, etc
+
```

Comment:
Add this into the bar that lives below the `navbar`; access information from the `activeBoard` that can be grabbed like so `const { activeBoard } = useKanban();`